slave/vagrant/fabfile.py

```diff
@@ -0,0 +1,56 @@
+from fabric.api import run, task, sudo, put, local
+from twisted.python.filepath import FilePath
+from StringIO import StringIO
+import yaml
+
+# WARNING: Although this code has been tested to produce a working slave
+# the production slave wasn't configured with this code.
+
+def configure_gsutil():
+    boto_config = FilePath(__file__).sibling('boto-config.in').getContent()
+    output = local('lpass show --notes "google-cert@build.clusterhq.com"',
+                   capture=True)
+    cert = yaml.safe_load(output.stdout)
```

Comment:
I don't understand this part.
LastPass notes for that account contains the cert (in binary form) I think....not yaml.
Perhaps LastPass notes need updating or perhaps you've got a different LastPass entry for `google-cert@build.clusterhq.com` .
I'm looking at the one in Shared-dev.clusterhq.com